src/node_zlib.cc

```diff
@@ -111,7 +111,12 @@ enum node_zlib_mode {
 
 struct CompressionError {
   CompressionError(const char* message, const char* code, int err)
-    : message(message), code(code), err(err) {}
+      : message(message)
+      , code(code)
+      , err(err) {
```

Comment:
```suggestion
      : message(message),
        code(code),
        err(err) {
```

To match with the common style of the codebase.